### PR TITLE
Add and update jobs

### PR DESCRIPTION
**Pull request by rabi**
- adds prow jobs for swift-operator
- changes the jobs for openstack-baremetal-operator

### CODE REVIEW

**Comment by rabi** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-swift-operator-main-functional

File: ci-operator/config/openstack-k8s-operators/swift-operator/openstack-k8s-operators-swift-operator-main.yaml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: functional
+  commands: export GOFLAGS= && make gotest
```
Comment (Sandeepyadav93) [inline ci-operator/config/openstack-k8s-operators/swift-operator/openstack-k8s-operators-swift-operator-main.yaml:43]: @rabi Some other operators use the below for functional


https://github.com/openshift/release/blob/cbb6ac043dcdfee8677b35fd386750b826b2fe6e/ci-operator/config/openstack-k8s-operators/ovn-operator/openstack-k8s-operators-ovn-operator-main.yaml#L56-L57

~~~
    export GOFLAGS=
    make test GINKGO_ARGS='--no-color'
~~~

Also, In some operators, we are using `gotest`  target and `test` target in others, On further checks I found `gotest` just calls the `test` target so I think that's okay. 

~~~
.PHONY: gotest
gotest: test
~~~

I think disabling ginkgo color output will help for logging, but we can do that in a follow-up.

**Review by Sandeepyadav93 — COMMENTED** [review]
Looks good, minor comment about disabling ginkgo color output but we can do that in follow up

**Pull request merged**